wrapper/Ada/README.md

```diff
@@ -0,0 +1,110 @@
+for the secondary stack. The GNAT User's Guide recommends
+avoiding the secondary stack using the restriction
+No_Secondary_Stack (see the GNAT configuration file gnat.adc
+which instructs compilation of the WolfSSL Ada binding under
+this restriction).
+
+Portability: The WolfSSL Ada binding makes no usage of controlled types
+and has no dependency upon the Ada.Finalization package.
+Lighter Ada run-times for embedded systems often have
+the restriction No_Finalization. The WolfSSL Ada binding has
+been developed with maximum portability in mind.
+
+Not only can the WolfSSL Ada binding be used in Ada applications but
+also SPARK applications (a subset of the Ada language suitable
+formal verification). To formally verify the Ada code in this repository
+open the client.gpr with GNAT Studio and then select
+SPARK -> Prove All Sources and use Proof Level 2.
+
+Summary of SPARK analysis
+=========================
+
+---------------------------------------------------------------------------------------------------------------
+SPARK Analysis results        Total        Flow   CodePeer                       Provers   Justified   Unproved
+---------------------------------------------------------------------------------------------------------------
+Data Dependencies                 2           2          .                             .           .          .
+Flow Dependencies                 .           .          .                             .           .          .
+Initialization                   15          15          .                             .           .          .
+Non-Aliasing                      .           .          .                             .           .          .
+Run-time Checks                  58           .          .    58 (CVC4 85%, Trivial 15%)           .          .
+Assertions                        6           .          .                      6 (CVC4)           .          .
+Functional Contracts             91           .          .                     91 (CVC4)           .          .
+LSP Verification                  .           .          .                             .           .          .
+Termination                       .           .          .                             .           .          .
+Concurrency                       .           .          .                             .           .          .
+---------------------------------------------------------------------------------------------------------------
+Total                           172    17 (10%)          .                     155 (90%)           .          .
+
+## Compiler and Build System installation
+
+### GNAT Community Edition 2021
+Download and install the GNAT community Edition 2021 compiler and studio:
+https://www.adacore.com/download
+
+Linux Install:
+
+```sh
+chmod +x gnat-2021-20210519-x86_64-linux-bin
+./gnat-2021-20210519-x86_64-linux-bin
+```
+
+```sh
+export PATH="/opt/GNAT/2021/bin:$PATH"
```

Comment:
I'd like the readme build instructions to be as close to copy and paste the commands into a terminal and it does a default build.

Can we add that gprclean is ran from the wrapper/Ada path. `cd wrapper/Ada`.

Reply:
Thanks for the feedback, I will update.